cmd/skaffold/app/cmd/config/set.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Skaffold Authors
+Copyright 2020 The Skaffold Authors
```

Comment:
we really shouldn't update these - please revert them, copyright is meant to show the creation time

Reply:
reverted on modified files, left 2020 on new files.